dispatch/static/manager/src/js/index.js

```diff
@@ -26,7 +28,14 @@ const store = createStore(
     loadingBar: loadingBarReducer,
     modal: modalReducer
   }),
-  applyMiddleware(thunk, router, promiseMiddleware(), loadingBarMiddleware())
+  applyMiddleware(
+    thunk,
+    router,
+    promiseMiddleware(),
+    loadingBarMiddleware({
+      promiseTypeSuffixes: ['PENDING', 'FULFILLED', 'REJECTED'],
```

Comment:
Do we need to add an `${ACTION_TYPE}_REJECTED` case to all the reducers as part of this change?